### PR TITLE
feat(compass): support pagination in SearchAssets

### PR DESCRIPTION
**Pull request by bsushmith**
* Add pagination support in SearchAssets API. `from` would be the offset and `size` would be the page size. The changes are backward compatible

### CODE REVIEW

**Comment by ravisuhag** [comment]
@bsushmith Let's discuss this please. It seems search is moving more and more towards list API. We should have clear responsibilities for list vs search.

**Comment by bsushmith** [comment]
Irrespective of the search vs list API's discussion - the current search returns back a list of results with no flexibility by caller on how many results they would want to show or how they would want to display. Pagination/cursor should be something that should have been already present. This only fixes that by adding a backward compatible change.

**Review by sudo-suhas — APPROVED** [review]
LGTM

File: odpf/compass/v1beta1/service.proto [hunk 1]
```diff
@@ -800,6 +800,16 @@ message SearchAssetsRequest {
     unique: true,
     ignore_empty: true
   }];
+
+  uint32 from = 7 [
```
Comment (sudo-suhas) [inline odpf/compass/v1beta1/service.proto:804]: Can we rename this to `offset` to keep it consistent with `GetAllAssetsRequest1`?

**Pull request merged**